tests/python/PyImportSortBearTest.py

```diff
@@ -1,29 +1,93 @@
 from bears.python.PyImportSortBear import PyImportSortBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
-PyImportSortBearTest = verify_local_bear(PyImportSortBear,
-                                         ('import os\nimport sys\n',
-                                          'import os\nimport sys\n'),
-                                         ('import sys\nimport os\n',
-                                          'import sys\nimport os\n'))
-
-PyImportSortBearConfigsTest = verify_local_bear(
-    PyImportSortBear,
-    ('from os import read\nfrom sys import *\n',),
-    ('from os import read\nfrom os import *\n',),
-    settings={'combine_star_imports': True})
```

Comment:
The decision to remove the tests should be a separate issue, and definitely a separate commit.

The reason for a separate issue is I'd prefer to see all existing tests pass with your PR , so I can feel safe there is no regression hiding in here , and then we talk about killing some of the tests after.

The utility of testing the tool is a reasonable debate to have.  IMO our users need to know our settings all work correctly.  And we need to know when a setting is broken by a major release of upstream, where they may delete functionality they do not like to maintain.  Or they change the setting name.  If we are not testing these things, the breakage is seen by our users.